src/sqlite3/statement.cr

```diff
@@ -45,6 +45,26 @@ class SQLite3::Statement < DB::Statement
     check LibSQLite3.bind_int(self, index, value ? 1 : 0)
   end
 
+  private def bind_arg(index, value : UInt8)
+    check LibSQLite3.bind_int(self, index, value.to_i)
+  end
+
+  private def bind_arg(index, value : UInt16)
+    check LibSQLite3.bind_int(self, index, value.to_i)
+  end
+
+  private def bind_arg(index, value : UInt32)
+    check LibSQLite3.bind_int(self, index, value)
```

Comment:
Shouldn't this use int64 conversion? Otherwise is casting it to int32 and the highest values would be stored as negatives i believe.

If bandwidth is over i'll try to add some tests to check for this.

Reply:
you are correct (though I didn't test it, since I'm only using UInt16 in my application). I just copy pasted it from the int ones and forgot that UInt32 needs to go into the 64 bit Int unlike the Int32 one. I've pushed a change to fix it.

As for tests, I unfortunately don't have bandwidth anymore since labor day is now over. I took a brief look at adding some more sample values in db_spec, but the complier complained about the types, so it seems like that'll have to tie in with crystal-db somewhere, but I think you all are probably better suited for making changes there. At the very least, my setup using Jennifer is now working with UInt16 and Int16. So I'd expect it be sound.